cpu/sam3/periph/rtt.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_sam3
+ * @ingroup     drivers_periph_rtt
+ *
+ * @note        The hardware RTT unit does neither support overflow interrupts
+ *              nor setting the counter value. For this, this RTT driver does
+ *              not implement those functions.
+ * @{
+ *
+ * @file
+ * @brief       Low-level RTT driver implementation
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "periph/rtt.h"
+
+#define ENABLE_DEBUG        (1)
+#include "debug.h"
+
+/* guard file to prevent file being build if no RTT was configured */
+#if RTT_NUMOF
```

Comment:
is this still needed?

Reply:
nope, its not. Will remove once I find time to actually get this driver working correctly...